docs/docs/ipc.md

```diff
@@ -107,8 +104,7 @@ Volume=/run/ipc/:/run/ipc/
 ```console
 [Unit]
 Description=Demo client service container
-Requires=ipc_server.socket
-After=ipc_server.socket
+
```

Comment:
Maybe have?
```suggestion
Requires=ipc_server.service
After=ipc_server.service

```

Having this dependency would allow it to mount the socket created by the server directly with:
```
Volume=/run/ipc.socket:/run/ipc.socket
```